src/Resolvers/UserResolver.php

```diff
@@ -23,6 +23,36 @@ class UserResolver implements \OwenIt\Auditing\Contracts\UserResolver
      */
     public static function resolve()
     {
-        return Auth::check() ? Auth::user()->getAuthIdentifier() : null;
+        return \Config::get('audit.morphable', false) ? static::resolveMorphable() : static::resolveSingle();
+    }
+
+    /**
+     * Resolves the user when not morphable.
+     *
+     * @return \Illuminate\Contracts\Auth\Authenticatable|null
+     */
+    private static function resolveSingle()
+    {
+        return Auth::check() ? Auth::user() : null;
+    }
+
+    /**
+     * Resolves the user when morphable.
+     *
+     * @return \Illuminate\Contracts\Auth\Authenticatable|null
+     */
+    private static function resolveMorphable()
```

Comment:
Like I stated earlier, we can default to this behaviour, even when not using a morph user.